test/expr/unary/keypath/keypath.swift

```diff
@@ -530,10 +530,10 @@ class X {
 }
 
 func testStaticKeyPathComponent() {
-  _ = \X.a // expected-error{{cannot refer to static member}}
-  _ = \X.Type.a // expected-error{{cannot refer to static member}}
-  _ = \X.b // expected-error{{cannot refer to static member}}
-  _ = \X.Type.b // expected-error{{cannot refer to static member}}
+  _ = \X.a // expected-error{{static member 'a' cannot be used on instance of type 'X'}}
+  _ = \X.Type.a
+  _ = \X.b // expected-error{{static member 'b' cannot be used on instance of type 'X'}}
+  _ = \X.Type.b
```

Comment:
We need a few tests here that flip the situation around and try to access instance member (property or subscript) on a metatype base.